utils/record/record_test.go

```diff
@@ -484,6 +484,18 @@ func TestNewAPIFactorySettings(t *testing.T) {
 		},
 	}
 
+	expectedSecrets := map[string][]byte{
+		"notification-secret": []byte("secret-value"),
+	}
+
+	notificationsSecret := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "argocd-notifications-secret",
```

Comment:
can we use argo rollouts secret name: argo-rollouts-notification-secret

Reply:
yeah, switched to the argo rollouts secret name.. it was mb..